GunShipAlpha/src/EnemySprites.java

```diff
@@ -41,10 +40,22 @@ public void draw(Graphics g, Point location) {
         g.drawImage(SPRITE, location.x, location.y, null);
     }
 }
- class TankSprite extends EnemySprites {
+ class MineSprite extends EnemySprites {
 
-     public TankSprite() {
-         super("/Images/Tank.png");
+     public MineSprite() {
+         super("/Images/Mine.png");
      }
 
 }
+class GinkerSprite extends EnemySprites {
+
```

Comment:
these subclasses, though, aren't really "pulling their weight" are they? 

they don't do anything but pass a different argument to the EnemySprites constructor, which you could just do where you call "new MineSprite()" and there's no need for the subclasses. 

Another observation is that you have an inconsistency between plural superclass EnemySprites and not in the subclass Mine (not Mines); I think it is probably singular? either way, they should be consistent when you have a subclass structure. this one shouldn't exist at all, i think, but something to keep in mind elsewhere.